.buildscript/deploy_snapshot.sh

```diff
@@ -16,7 +16,6 @@ if [ "$CIRCLE_PROJECT_REPONAME" != "$REPO" ]; then
 elif [ "$CIRCLE_PROJECT_USERNAME" != "$USERNAME" ]; then
   echo "Skipping snapshot deployment: wrong owner. Expected '$USERNAME' but was '$CIRCLE_PROJECT_USERNAME'."
 elif [ "$CIRCLE_JDK_VERSION" != "$JDK" ]; then
-  # $CIRCLE_JDK_VERSION must be manually set in circle.yml
```

Comment:
Oh interesting, I missed this.

Looks like even analytics-android failed. https://circleci.com/gh/segmentio/analytics-android/1008

Reply:
Probably ok for now as we don't need snapshots here, but let's hold off on the other integrations until then.

Reply:
Tracking in https://segment.atlassian.net/browse/LIB-160